ts/src/apps/github/constants.ts

```diff
@@ -61,14 +131,38 @@ export const GITHUB_ALLOWED_PATHS: TAllowedPaths = {
     },
   },
   '/repos/{owner}/{repo}': {
-    GET: {},
-    PATCH: {},
-    DELETE: {},
+    GET: {
+      override_options: {
+        ...repoOwnerCommonOptions,
+      },
+    },
+    PATCH: {
+      override_options: {
+        ...repoOwnerCommonOptions,
+      },
+    },
+    DELETE: {
+      override_options: {
+        ...repoOwnerCommonOptions,
+      },
+    },
   },
   '/repos/{owner}/{repo}/contents/{path}': {
-    GET: {},
-    PUT: {},
-    DELETE: {},
+    GET: {
+      override_options: {
+        ...repoOwnerCommonOptions,
+      },
+    },
+    PUT: {
+      override_options: {
+        ...repoOwnerCommonOptions,
+      },
+    },
+    DELETE: {
+      override_options: {
+        ...repoOwnerCommonOptions,
```

Comment:
why not just `override_options: repoOwnerCommonOptions` ?

Reply:
Decided to leave it like this, in case additional options would be added in the future.